packages/rainbowkit/src/components/ConnectButton/ConnectButton.tsx

```diff
@@ -39,6 +40,14 @@ export function ConnectButton({
 }: ConnectButtonProps) {
   const chains = useRainbowKitChains();
   const connectionStatus = useConnectionStatus();
+  const [isClient, setIsClient] = useState(false);
+
+  const i18n = useContext(I18nContext);
+
+  // Quick fix to avoid hydration errors with next-js on brave browser
+  useEffect(() => {
+    setIsClient(true);
+  }, []);
```

Comment:
Added this avoid rehydration error from nextjs on other brave browser when not providing `locale` prop. Just simple hack.

Reply:
@KosmosKey What issue does this resolve? Is this a SSR issue?

Reply:
Yeah it's an SSR issue. It will look like this:

![Uploading image.png…]()

Reply:
@DanielSinclair I wanted to remove this early today, but dev people will have hard time on this an open ticket which might lead to more github issues.

I can explain what is happening so we might keep it as is:
- When doing browser detection in SSR next.js has `Connect Wallet` while running build
- On CSR the language detection updates the locale to be something like `वॉलेट को कनेक्ट करें`

Next.js is then getting confused but this won't affect prod but while devs are working it might impact them.